tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -349,20 +349,6 @@ class InternalCallMapHandlerTest extends TestCase
         'intltz_get_display_name',
         'iteratoriterator::__construct',
         'jsonexception::__construct',
-        'ldap_compare' => ['8.0'],
```

Comment:
Many of these had changes from php 7.2 and 8.0 combined into a single php 8.1 delta for some reason.